publishable/lang/tr/voyager.php

```diff
@@ -94,74 +94,82 @@
     ],
 
     'login' => [
-        'loggingin'    => 'Giriş yap',
-        'signin_below' => ' Below: oturum aç',
-        'welcome'      => "Voyager a hoş geldiniz , Laravel'in aranan yönetim paneli",
+        'loggingin'    => 'Giriş Yap',
+        'signin_below' => 'Oturum Aç',
+        'welcome'      => "Voyager'a hoş geldiniz. Laravel'in Aranan Yönetim Paneli",
     ],
 
     'profile' => [
         'avatar'        => 'Avatar',
-        'edit'          => 'Profilimi düzenle',
-        'edit_user'     => 'Kullanıcıyı düzenle',
+        'edit'          => 'Profilimi Düzenle',
+        'edit_user'     => 'Kullanıcıyı Düzenle',
         'password'      => 'Şifre',
-        'password_hint' => 'Aynı şifre ise boş bırakın',
+        'password_hint' => 'Aynı kalması için boş bırakın',
         'role'          => 'Rol',
         'user_role'     => 'Kullanıcı Rolü',
     ],
     'settings' => [
-        'usage_help'           => 'Her ayarın değerini sitenizdeki herhangi bir yerinden çağırabilirisiniz',
+        'usage_help'           => 'Her ayarın değerini sitenizdeki herhangi bir yerden çağırabilirisiniz',
         'save'                 => 'Ayarları Kaydet',
         'new'                  => 'Yeni Ayarlar',
         'help_name'            => 'Ayar Adı ör: Yönetici paneli',
         'help_key'             => 'Ayar anahtarı ör: yonetici_paneli',
         'help_option'          => '(İsteğe bağlı, only applies to certain types like dropdown box or radio button)',
-        'add_new'              => 'Yeni Ayar ekle',
-        'delete_question'      => ' :setting Bu ayarı silmek istediğine emin misin?',
-        'delete_confirm'       => 'Evet , Bu Ayarı Sil',
-        'successfully_created' => 'Başarıyla Ayar  Oluşturuldu',
-        'successfully_saved'   => 'Başarıyla Ayar kaydedildi ',
-        'successfully_deleted' => 'Başarıyla Ayar silindi ',
+        'add_new'              => 'Yeni Ayar Ekle',
+        'delete_question'      => ' :setting ayarını silmek istediğine emin misin?',
+        'delete_confirm'       => 'Evet, Bu Ayarı Sil',
+        'successfully_created' => 'Başarıyla Ayar Oluşturuldu',
+        'successfully_saved'   => 'Başarıyla Ayar Kaydedildi ',
+        'successfully_deleted' => 'Başarıyla Ayar Silindi ',
         'already_at_top'       => 'Zaten Listenin en üstünde',
         'already_at_bottom'    => 'Zaten listenin en altında',
+        'key_already_exists'   => ':key anahtarı zaten mevcut',        
```

Comment:
Hi! Can you remove unwanted spaces?

Reply:
StyleCI will auto-fix these issues once it's merged.  I'm not very worried about failed StyleCI checks.

Reply:
Thanks. I don't know.

Reply:
No worries.